src/js/components/Anchor/README.md

```diff
@@ -1,7 +1,6 @@
 ## Anchor
-A text link. We have a separate component from the browser
-base so we can style it. You can either set the icon and/or label properties
-or just use children.
+A text link.
+###### We have a separate component from the browser base so we can style it. You can either set the icon and/or label properties or just use children.
```

Comment:
Why is this an level six heading?

Reply:
to show in a smaller font size

Reply:
But semantically, it's not a header. It should be a paragraph, since it might have Markdown details within it and span multiple lines. I would expect it in the README to just be the paragraphs following the description.